drivers/google/google.go

```diff
@@ -260,20 +271,21 @@ func (d *Driver) Remove() error {
 	return c.deleteDisk()
 }
 
-// Restart deletes and recreates the GCE instance, keeping the disk.
 func (d *Driver) Restart() error {
+	return nil
+}
+
+// Kill deletes the GCE instance, but keeps the disk.
```

Comment:
Passing off to `Stop` seems acceptable to me.  `Remove` is the method responsible for deleting and cleaning up the instance.